shared/desktop/app/index.js

```diff
@@ -26,6 +27,10 @@ function start() {
       if (mainWindow) {
         mainWindow.show(true)
       }
+      if (isWindows) {
+        // Wasn't showing on Windows without this
+        showMainWindow()
```

Comment:
After more looking, I think the only difference is that the `showMain` handler also does `mainWindow.window && mainWindow.window.focus()` - but I can't quite test that because of broken Windows package